common/utilities/convertStringToBoolean.js

```diff
@@ -0,0 +1,8 @@
+export default string => {
```

Comment:
This feels odd, not sure if you can think of a better way?

Reply:
it does feel odd because if(something) { return true } is nearly always a boolean expression.

Your function also has an inconsistent return type (sometimes returns boolean, sometimes a string) which is generally not recommended.

I'd go with `return string === 'true'`